tests/apollo/test_skvbc_reconfiguration.py

```diff
@@ -841,11 +842,55 @@ async def test_wedge_command_with_state_transfer(self, bft_network):
         for r in late_replicas:
             await bft_network.wait_for_state_transfer_to_stop(initial_prim,
                                                               r,
-                                                              stop_on_stable_seq_num=False)
-        await self.try_to_unwedge(bft_network=bft_network, bft=False, restart=False)
+                                                              stop_on_stable_seq_num=True)
         for i in range(100):
             await skvbc.send_write_kv_set()
 
+    @with_trio
+    @with_bft_network(start_replica_cmd, selected_configs=lambda n, f, c: n == 7, publish_master_keys=True)
+    async def test_wedge_command_with_state_transfer_with_write(self, bft_network):
+        """
+            This test checks that a replica that received the super stable checkpoint and further client writes via the state transfer mechanism
```

Comment:
Nit: Could you please correct below comment? 

` further client writes via the state transfer mechanism`

Reply:
Please have single commit.
I see 5 commits.

Reply:
Fixed